src/_ert/forward_model_runner/client.py

```diff
@@ -26,107 +22,139 @@ class ClientConnectionClosedOK(Exception):
 
 
 class Client:
-    DEFAULT_MAX_RETRIES = 10
-    DEFAULT_TIMEOUT_MULTIPLIER = 5
-    CONNECTION_TIMEOUT = 60
+    DEFAULT_MAX_RETRIES = 5
+    DEFAULT_ACK_TIMEOUT = 5
+    _receiver_task: asyncio.Task[None] | None
 
     def __enter__(self) -> Self:
+        self.loop.run_until_complete(self.__aenter__())
```

Comment:
Maybe we can remove all of the synced wrappers of the async methods?

Reply:
Maybe. I think the synced version if now only used in tests.

Reply:
I have made some exploration and the issue that the reporter tests are still fully synced and thus we need to preserve both (async and sync) versions alive.